### DISCUSSION TITLE
Automation only if specific column changed

### DISCUSSION THREAD

**Discussion by sebclemens** [Budibase/budibase#10033] [Help] [answered]
I wanna send an automation mail only if the value of column "participation" changed. At the moment I send an mail if anything updated, I don't know how to do otherwise.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @sebclemens,

You can use the conditional step in this case: https://docs.budibase.com/docs/conditions-1, check to see if the specific field you are talking about is updated, if it is send the email. If not stop the automation.

I hope this helps!

**Reply by sebclemens** [reply]
I don't get it. There I can just compare the value of participitation to "yes" or "no", but not if it changed.

**Reply by ConorWebb96** [reply]
Hey @sebclemens,

Sorry about the confusion! After looking into this a bit further you can't add conditions related to specific field updates within a row. 
This would only work if you have a static value e.g. Boolean to check against.

However, I have created an enhancement request for this. This can be found here: https://github.com/Budibase/budibase/issues/10035

Again, apologies for the misunderstanding!
